images/build/Dockerfiles/azureFunctions.JamStack.Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG PARENT_DEBIAN_FLAVOR
-FROM oryxdevmcr.azurecr.io/public/oryx/build:github-${PARENT_DEBIAN_FLAVOR} AS main
 ARG DEBIAN_FLAVOR
+FROM oryxdevmcr.azurecr.io/public/oryx/cli-${DEBIAN_FLAVOR}:${PARENT_DEBIAN_FLAVOR} AS main
```

Comment:
Just calling out that we'd need to change this repo if we decide to go with the `oryx/cli:debian-{flavor}-{tag}` format before merging this PR.